qa/workunits/rados/test_rados_tool.sh

```diff
@@ -258,12 +258,22 @@ run_expect_nosignal "$RADOS_TOOL" --pool "$POOL" bench 1 seq
 set -e
 
 OBJ=test_rados_obj
+p=`uuidgen`
```

Comment:
`p` is not referenced anywhere else other than `test_rmobj()`. could move it into that function.